lib/expo/po/tokenizer.ex

```diff
@@ -52,9 +52,9 @@ defmodule Expo.PO.Tokenizer do
     * `{:str, 6, "foo"}`
 
   """
-  @spec tokenize(binary) :: {:ok, [token]} | {:error, pos_integer, binary}
-  def tokenize(str) do
-    tokenize_line(str, _line = 1, _tokens_acc = [])
+  @spec tokenize(binary, boolean()) :: {:ok, [token]} | {:error, pos_integer, binary}
+  def tokenize(str, strip_meta \\ false) do
```

Comment:
To avoid boolean arguments, I would forward some options here and accept options here too. Something like:

```elixir
tokenize(str, Keyword.take(options, [:strip_meta]))

# and here:
strip_meta? = Keyword.get(options, :strip_meta, false)
```